lib/signed-xml.js

```diff
@@ -861,7 +869,7 @@ SignedXml.prototype.getKeyInfo = function(prefix) {
 
   if (this.keyInfoProvider) {
     res += "<" + currentPrefix + "KeyInfo>"
-    res += this.keyInfoProvider.getKeyInfo(this.signingKey, prefix)
+    res += this.keyInfoProvider.getKeyInfo(this.signingCert, prefix)
```

Comment:
Potential breaking change, as existing users will be relying on `signingKey` to generate the key info. What are you trying to do here?